app/controllers/vm_common.rb

```diff
@@ -156,22 +156,23 @@ def vm_selected
   end
 
   def launch_html5_console
-    proto = request.ssl? ? 'wss' : 'ws'
+    prefix = request.ssl? ? 'wss' : 'ws'
```

Comment:
why rename `proto` to `prefix`?

it's a protocol so the new one is not a better name